artifactory/users.go

```diff
@@ -65,10 +65,8 @@ func (d DeleteAPIKey) String() string {
 }
 
 // GetAll returns a list of all users.
-//
-// Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetUsers
```

Comment:
Unfortunately, this endpoint is currently undocumented... 

**Note**: I've created a support ticket with JFrog inquiring if they can update the docs to reflect the functionality.